interfaces/mount/change_test.go

```diff
@@ -174,3 +191,48 @@ func (s *changeSuite) TestNeededChangesSmartEntryComparison(c *C) {
 		{Entry: mount.Entry{Dir: "/a/b/c"}, Action: mount.Mount},
 	})
 }
+
+// Change.Perform calls the mount system call.
+func (s *changeSuite) TestPerformMount(c *C) {
+	chg := &mount.Change{Action: mount.Mount, Entry: mount.Entry{Name: "source", Dir: "target", Type: "type"}}
+	c.Assert(chg.Perform(), IsNil)
+	c.Assert(s.sys.Calls(), DeepEquals, []string{`mount "source" "target" "type" 0 ""`})
+}
+
+// Change.Perform returns errors from mount system call
+func (s *changeSuite) TestPerformMountError(c *C) {
+	s.sys.InsertFault(`mount "source" "target" "type" 0 ""`, errTesting)
+	chg := &mount.Change{Action: mount.Mount, Entry: mount.Entry{Name: "source", Dir: "target", Type: "type"}}
+	c.Assert(chg.Perform(), Equals, errTesting)
+	c.Assert(s.sys.Calls(), DeepEquals, []string{`mount "source" "target" "type" 0 ""`})
+}
+
+// Change.Perform returns errors from bad flags
+func (s *changeSuite) TestPerformMountOptionError(c *C) {
+	chg := &mount.Change{Action: mount.Mount, Entry: mount.Entry{Name: "source", Dir: "target", Type: "type", Options: []string{"bogus"}}}
+	c.Assert(chg.Perform(), ErrorMatches, `unsupported mount option: "bogus"`)
+	c.Assert(s.sys.Calls(), HasLen, 0)
+}
+
+// Change.Perform calls the unmount system call.
+func (s *changeSuite) TestPerformUnmount(c *C) {
+	chg := &mount.Change{Action: mount.Unmount, Entry: mount.Entry{Name: "source", Dir: "target", Type: "type"}}
+	c.Assert(chg.Perform(), IsNil)
+	// The flag 8 is UMOUNT_NOFOLLOW
+	c.Assert(s.sys.Calls(), DeepEquals, []string{`unmount "target" 8`})
+}
+
+// Change.Perform returns errors from unmount system call
+func (s *changeSuite) TestPerformUnountError(c *C) {
+	s.sys.InsertFault(`unmount "target" 8`, errTesting)
```

Comment:
Maybe a comment like above: `// The flag 8 is UMOUNT_NOFOLLOW` ? I also wonder if there is a way to make this easier to read in the tests. Not sure how hard it would be to resolve the flags in the recorder to symbolic names?

Reply:
I'll look into this.

Reply:
Done, this is symbolic now